### PR TITLE
Fix cropper by using classic firebase storage

### LINKED ISSUE #2395: [Release 1.3] Cropper / Image Upload not working on account
Page Profile & Org

### PR DESCRIPTION
**Pull request by ioaNikas**
Fix cropper by using classic firebase storage instead of angular fire one

close #2395

### CODE REVIEW

File: libs/ui/src/lib/media/cropper/cropper/cropper.component.ts [hunk 1]
```diff
@@ -168,18 +167,20 @@ export class CropperComponent implements ControlValueAccessor {
...
-      })
+  async goToShow() {
+    // TODO#2451: switch back to angularFire storage ref when bug on getMetadata() is fixed.
+    const ref = firebaseStorage().ref(`${this.folder}/${this.fileName}`);
+    const metadata = await ref.getMetadata();
+    this.url$ = this.ref.getDownloadURL().pipe(
+      catchError(err => of(''))
+    )
+
+    this.url$.subscribe(url => this.uploaded(createImgRef({
```
Comment (Hugo-Hontans) [inline libs/ui/src/lib/media/cropper/cropper/cropper.component.ts:178]: Where do you unsubscribe ?

File: env/__ngcc_entry_points__.json [hunk 2]
```diff
@@ -1 +1 @@
-{"ngccVersion":"9.1.0","configFileHash":"87c535c3ce0eac2a54c246892e0e21a1","lockFileHash":"43f38fbc06a7acb82b98b4e69e677d8a","entryPointPaths":[]}
\ No newline at end of file
+{"ngccVersion":"9.1.0","configFileHash":"87c535c3ce0eac2a54c246892e0e21a1","lockFileHash":"b1eca11836c2d36d33895cc7a5dca236","entryPointPaths":[]}
```
Comment (Hugo-Hontans) [inline env/__ngcc_entry_points__.json:1]: What is this key ?

Comment (ioaNikas) [reply env/__ngcc_entry_points__.json:1]: I have no clue !

Comment (ioaNikas) [reply env/__ngcc_entry_points__.json:1]: Who knows ?

**Pull request merged**